bw2io/backup.py

```diff
@@ -34,15 +34,25 @@ def backup_data_directory():
         tar.add(projects.dir, arcname=os.path.basename(projects.dir))
 
 
-def backup_project_directory(project: str):
+def backup_project_directory(
+    project: str, timestamp: Optional[bool] = True, dir_backup: Optional[str] = None
```

Comment:
Can you talk about why `dir_backup` is a `str` and not a `pathlib.Path`?

Reply:
I considered that, but I was trying may as few changes to the code as possible. Since the rest was using os.path, I stuck with that in my changes. Also, I wanted to keep the user side the same, not to break anything.

I remember you saying sometime that you wanted to move to everything using Path, I think.
I adapted it now. To only use pathlib

The user would always give a string, right? Maybe not. I added:
`dir_backup: Optional[Union[str, Path]] = None`
To allow both inputs